std_msgs/Byte number
std_msgs/Byte condition
---
